Gopkg.lock

```diff
@@ -33,14 +33,6 @@
   pruneopts = "T"
   revision = "de5bf2ad457846296e2031421a34e2568e304e35"
 
-[[projects]]
-  digest = "1:9187ad53cd8b43cc82d1615ca6319503735fd87cd2b4dbeb80c57831e9affd79"
-  name = "github.com/caicloud/serving-controller"
-  packages = ["pkg/consts"]
-  pruneopts = "T"
-  revision = "bec825cac979e73109541ca7518dd6b96ce9e68a"
-  version = "v0.0.1-rc.5"
```

Comment:
LGTM, but your PR just reminds me that maybe we should includes Gopkg* into prow, for example we upgrades some reps

Reply:
Actually, I am wondering why it passes the CI test. It should raise an error during dep ensure.

Reply:
i did this in  https://github.com/kubeflow/katib/pull/512/files . It was failing for me

Reply:
Yeah, it should fail in CI, too. I am not sure why it passed.

Reply:
Anyway, I will add gopkg in prow config to trigger CI in this PR.

Reply:
Addressed